asteroid/data/dampvsep_dataset.py

```diff
@@ -0,0 +1,225 @@
+        samples_per_track (int, optional):
+            Number of samples yielded from each track, can be used to increase
+            dataset size, defaults to ``1``.
+        random_segments (boolean, optional): Enables random offset for track segments.
+        sample_rate (int, optional): Sample rate of files in dataset.
+            Default 16000 Hz
+        segment (float, optional): Duration of segments in seconds,
+            Defaults to ``None`` which loads the full-length audio tracks.
+        num_workers (int, optional): Number of workers.
+            Default to ``None`` which utilise all available workers
+        norm (Str, optional): Type of normalisation to use. Default to ``None``
+            * ``'song_level'`` use mixture mean and std.
+            * ```None``` no normalisation
+        source_augmentations (Composite, optional): Default to ``None``
+        mixture (str, optional): Whether to use the original mixture with non-linear effects
+                                or remix sources. Default to original.
+            * ``'remix'`` for use addition to remix the sources.
+            * ``'original'`` for use the original mixture.
+    """
+
+    dataset_name = "DAMP-VSEP"
+
+    def __init__(
+        self,
+        root_path,
+        task,
+        split="train_singles",
+        samples_per_track=1,
+        random_segments=False,
+        sample_rate=16000,
+        segment=None,
+        num_workers=None,
+        norm=None,
+        source_augmentations=None,
+        mixture="original",
+    ):
+
+        self.sample_rate = sample_rate
+        self.num_workers = cpu_count() if num_workers is None else num_workers
+
+        self.root_path = Path(root_path).expanduser()
+        # Task detail parameters
+        assert task in ["enh_vocal", "enh_both"], "Task should be one of 'enh_vocal','enh_both'"
+        assert mixture in ["remix", "original"], "Mixture should be one of 'remix', 'original'"
+
+        self.task = task
+        if task == "enh_vocal":
+            self.target = ["vocal"]
+        elif task == "enh_both":
+            self.target = ["vocal", "background"]
+
+        self.split = split
+        self.tracks = self.get_tracks()
+        self.perf_key = [*self.tracks]  # list of performances keys
+
+        self.samples_per_track = samples_per_track
+        self.random_segments = random_segments
+        self.sample_rate = sample_rate
+        self.segment = segment
+        self.norm = norm
+        self.source_augmentations = source_augmentations
+        self.mixture = mixture
+        if self.mixture == "original" and self.split == "train_english":
+            raise Exception("The 'train_english' train can only accept 'remix' mixture.")
+
+    def __len__(self):
+        return len(self.tracks) * self.samples_per_track
+
+    def _load_audio(self, path, start=0.0, duration=None, scaler=None, mean=0.0, std=1.0):
+        x, _ = librosa.load(
```

Comment:
Import librosa here, rather than on the top. We don't have explicit deps for it.